core/executor.py

```
@@ -413,37 +618,103 @@ def act_phase(self, action: Dict) -> Any:
+            result = None
+            last_error = None
+
+            for attempt in range(max_retries):
+                try:
+                    result = (
+                        self.tool_registry.execute(tool_name, arguments)
+                        if self.tool_registry
+                        else f"Tool registry not available: {tool_name}"
+                    )
+
+                    # 如果执行成功，返回结果
```

Ruff (0.15.9)

[warning] Comment contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF003)